inlong-manager/manager-test/src/main/resources/h2/apache_inlong_manager.sql

```diff
@@ -785,7 +785,9 @@ VALUES ('audit_sdk_collect', 'SDK', 0, '1'),
        ('audit_sort_hbase_input', 'HBASE', 0, '19'),
        ('audit_sort_hbase_output', 'HBASE', 1, '20'),
        ('audit_sort_doris_input', 'DORIS', 0, '21'),
-       ('audit_sort_doris_output', 'DORIS', 1, '22');
+       ('audit_sort_doris_output', 'DORIS', 1, '22'),
```

Comment:
Please add the `mysql input and output`, to keep the same with another SQL file, thanks.